lib/rmt/config.rb

```diff
@@ -3,15 +3,34 @@
 
 Config.setup do |config|
   config.merge_nil_values = false
+
+  unless ENV['RAILS_ENV'] == 'test'
+    config.schema do
+      required(:scc).schema do
+        required(:username).filled
+        required(:password).filled
```

Comment:
As I've mentioned in my previous comment, SCC username and password actually aren't always required. Yast RMT module allows skipping setting username and password [as of late](https://github.com/yast/yast-rmt/pull/44), which is required for airgap setups. (Also not having to set SCC username and passwords might come handy for openSUSE use cases)

Reply:
Understood! So I'll remove the `filled` requirements from the schema.

Reply:
I think that will fix the tests :-)